examples/version-migration/tree-shim/src/model/newTreeInventoryListController.ts

```diff
@@ -128,27 +122,7 @@ export class NewTreeInventoryListController extends EventEmitter implements IInv
 		// 3. On all loads, gets an (untyped) view of the data (the contents can't be accessed directly from the sharedTree).
 		// Then the root2() call applies a typing to the untyped view based on our schema.  After that we can actually
 		// reach in and grab the inventoryItems list.
-		this._inventoryItemList = this._tree.schematizeOld({
-			initialTree: {
-				inventoryItemList: {
-					// TODO: The list type unfortunately needs this "" key for now, but it's supposed to go away soon.
-					"": [
-						{
-							id: uuid(),
-							name: "nut",
-							quantity: 0,
-						},
-						{
-							id: uuid(),
-							name: "bolt",
-							quantity: 0,
-						},
-					],
-				},
-			},
-			allowedSchemaModifications: AllowedUpdateType.None,
-			schema,
-		}).root.inventoryItemList;
+		this._inventoryItemList = this._tree.schematize(treeConfiguration).root.inventoryItemList;
```

Comment:
I know I missed this review, but it would be good if you can update the comment above to explain to a tree-learner what's happening here under the new API.